assets/queries/k8s/psp_allows_sharing_host_ipc/query.rego

```diff
@@ -0,0 +1,16 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    document.kind == "PodSecurityPolicy"
+
```

Comment:
Add metadata information to improve our searchKey

metadata := document.metadata